<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSerie">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="Chapter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Chapter</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="URL">
      <property name="text">
       <string>URL (Must contain [Chapter] and [Image])</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TextURL"/>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="ChapterValue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="VolumeValue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>250</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="BoxWeeklyPDF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Weekly PDF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
